terms.html

```diff
@@ -80,15 +81,13 @@
+<p><span class="section-number">28</span> If you are using any other pricing plan and we have announced changes to our terms and conditions, you may terminate your contract with us by giving us notice in writing within 21 days of the date on which the announcement was made.  Under these circumstances, your right of access to PDF Tables will terminate on the date the new terms and conditions come into force. You will not be entitled to a refund in respect of any outstanding credit or other money paid to us.</p>
 <h2>Termination</h2>
-<p><span class="section-number">29</span> If you are using this service under the terms of our free pricing plan, you may terminate your use of the service at any time by closing your account from within the account options page.</p>
-<p><span class="section-number">30</span> If you are using this service under the terms of a paid pricing plan, service is provided on the basis of a monthly rolling contract with payment in advance unless there is a specific written agreement between us and you to the contrary.  You may cancel the service at any time by giving one month's notice of your intention to do so.</p>
-<p><span class="section-number">31</span> There are unfortunately circumstances, which we think will be extremely unlikely to occur, where we may have to terminate your access - for example where we have been ordered to do so by a competent authority and we are not permitted to tell you why. For this reason, we reserve the right to terminate your access to the service at any time without the need to give you a reason, although we will try to give one if possible.  In the unlikely event that we terminate your access in this way, we will refund any money paid for the service on a pro rata basis within 30 days.</p>
+<p><span class="section-number">29</span>You may terminate your use of PDF Tables at any time by sending a message to pdftables@scraperwiki.com. We will not refund any outstanding credit to your account if you terminate it in this way.</p>
+<p><span class="section-number">31</span> There are unfortunately circumstances, which we think will be extremely unlikely to occur, where we may have to terminate your access - for example where we have been ordered to do so by a competent authority and we are not permitted to tell you why. For this reason, we reserve the right to terminate your access to the service at any time without the need to give you a reason, although we will try to give one if possible.  In the unlikely event that we terminate your access in this way, we will refund any outstanding credit within 30 days.</p>
```

Comment:
Yes, that's true. Worse than simply pro rata for a monthly subscription.

However, being able to take someone's money for a service and then not give them the service while keeping the money not only seems unattractive it might actually be unlawful in certain circumstances. Do you want to amend this to "we will not". I've softened it a little in draft.